src/HowToLearngraph.tsx

```diff
@@ -0,0 +1,28 @@
+import { AppBar, Toolbar } from '@mui/material'
+import Image from 'mui-image'
+//import { LearngraphLOGO } from './GraphManager/components/HeaderBar'
+import { useNavigate } from 'react-router-dom'
+
+export const HowToLearngraph = () => {
+    const navigate = useNavigate();
+    return (
+        <>
+                <AppBar position='static'>
```

Comment:
There have been some changes on the main branch. If you want we can take a look at it together! :)
After merging the header bar can now be imported as <NavigationWithContent />, and must no longer be copied to every page, see https://github.com/suxatcode/learn-graph-frontend/blob/e9fed081c17f53bb574f3aa21d39e121cc602b55/src/GraphManager/GraphManager.tsx#L29 for example usage. You won't need the `search` and property, you can just omit it.
As you can see here https://github.com/suxatcode/learn-graph-frontend/blob/e9fed081c17f53bb574f3aa21d39e121cc602b55/src/Navigation.tsx#L226 the LOGO is already in the NavigationWithContent component by default.